### PR TITLE
Add S2 L1 & ARD products

### PR DESCRIPTION
**Pull request by jeremyh**
... used in the s2_test database

This also fixes the previous CI validation warnings, and re-enables the stricter mode of checking that was disabled in  c7035cdf1f81d40a9243e16f85b4255257da2265

### CODE REVIEW

**Comment by jeremyh** [comment]
Future changes will be separate PRs. I think we're close enough to merge this now.

**Comment by dunkgray** [comment]
Hi Jeremy,
This looks good.
Could you push this up to a branch on the main repo so I can suggest some changes?

Cheers
Duncan
________________________________

**Comment by jeremyh** [comment]
> Hi Jeremy, This looks good. Could you push this up to a branch on the main repo so I can suggest some changes? Cheers Duncan
> […](#)

You should be able to add suggestions here on the PR? Or do you want to push your own commits?

**Comment by dunkgray** [comment]
I can't do edits to the code in the PR currently.
I think I would be able to if these changes where on a branch of the main repo.
________________________________

**Comment by dunkgray** [comment]
Hi Jeremy,
 Could you push this up to a branch on the main repo?

Cheers
Duncan
________________________________

File: digitalearthau/config/eo3/products/l1_s2a.odc-product.yaml [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+name: esa_s2am_level1_0
+description: ESA Sentinel 2A Level 1
```
Comment (dunkgray) [inline digitalearthau/config/eo3/products/l1_s2a.odc-product.yaml]:
```suggestion
description: ESA Sentinel 2A Level 0
```

File: digitalearthau/config/eo3/products/l1_s2a.odc-product.yaml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+metadata:
+  product:
+    name: esa_s2am_level1_0
+  properties:
+     eo:platform: sentinel-2a
+     eo:instrument: MSI
+     odc:product_family: level1
+     odc:producer: esa.int
+     # ? dea:product_maturity: final
```
Comment (dunkgray) [inline digitalearthau/config/eo3/products/l1_s2a.odc-product.yaml]:
```suggestion

```

File: digitalearthau/config/eo3/products/l1_s2b.odc-product.yaml [hunk 3]
```diff
@@ -0,0 +1,73 @@
+---
+name: esa_s2bm_level1_0
+description: ESA Sentinel 2B Level 1
```
Comment (dunkgray) [inline digitalearthau/config/eo3/products/l1_s2b.odc-product.yaml]:
```suggestion
description: ESA Sentinel 2B Level 0
```

File: digitalearthau/config/eo3/products/l1_s2b.odc-product.yaml [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+metadata:
+  product:
+    name: esa_s2bm_level1_0
+  properties:
+     eo:platform: sentinel-2b
+     eo:instrument: MSI
+     odc:product_family: level1
+     odc:producer: esa.int
+     # ? dea:product_maturity: final
```
Comment (dunkgray) [inline digitalearthau/config/eo3/products/l1_s2b.odc-product.yaml]:
```suggestion

```

**Review by dunkgray — APPROVED** [review]
Thanks for you help on adding suggestions

**Pull request merged**